### PR TITLE
Add a concise test report with just the errors

### LINKED ISSUE #4124: Add a concise test report with just the errors
## Scope & Context

Running test on the different folders.

## Technical inputs

We would like to have a clear and concise report of the failed tests in the different apps, listing only the test files in error and their path at the end of the logs, so we don't have to scroll through the logs to find out.

Hint : see if we can configure jest report to only print file paths for failed tests.

### PR DESCRIPTION
**Pull request by gitstart-twenty**
### Description
Add a concise test report with just the errors

### Refs
#4124

##### Case: No failed tests found.
TWNTY-4124 Test Errors

##### Case: Found failing tests.
TWNTY-4124 Test Report

Fixes #4124

### CODE REVIEW

**Comment by gitstart-twenty** [comment]
Hey @lucasbordeau,
What do you think of this approach?

**Comment by gitstart-twenty** [comment]
Hey @charlesBochet,
Should we proceed to update the reporters for the other apps?

**Review by charlesBochet — COMMENTED** [review]
I think we want to keep the fail detail but remove all the output from the console

**Review by charlesBochet — COMMENTED** [review]
@gitstart-twenty I think what we would like:
- remove the console output flood (right now that's the main issue)
- keep the full test output

Maybe we could leverage something like this:
https://stackoverflow.com/questions/44467657/better-way-to-disable-console-inside-unit-tests

**Comment by gitstart-twenty** [comment]
> @gitstart-twenty I think what we would like:
> 
> * remove the console output flood (right now that's the main issue)
> * keep the full test output
> 
> Maybe we could leverage something like this: https://stackoverflow.com/questions/44467657/better-way-to-disable-console-inside-unit-tests

Alright

**Comment by gitstart-twenty** [comment]
Hey @charlesBochet,

We already have an entry for `setupFilesAfterEnv` with value `['./src/setupTests.ts']` in `packages/twenty-front/jest.config.ts`. Much as we can disable certain types of logs by making these changes to this file:

```ts
global.console = {
  ...console,
  // uncomment to ignore a specific log level
  log: jest.fn(),
  debug: jest.fn(),
  info: jest.fn(),
  // warn: jest.fn(),
  // error: jest.fn(),
};
```

When an error is thrown(even if it doesn't fail the test), we'll still have a console output flood with long stack traces. The only way to stop this is by setting the silent flag to true in `packages/twenty-front/jest.config.ts`

```ts
export default {
  silent: true,
  // ...
}
```

The output will look like this with this flag set to true:
Screenshot 2024-03-05 at 12 09 38

This is much cleaner but however, please note that the coverage report is disabled in the above test report i.e the tests were run with `jest --coverage=false`. Running the tests with just `jest` will still have the coverage report taking up a lot of the output:

Screenshot 2024-03-05 at 12 11 11

**Comment by charlesBochet** [comment]
Looks great! It's still weird that the Summary of failed tests is showing in coverage file list but it's not a big deal

**Comment by gitstart-twenty** [comment]
> Looks great! It's still weird that the Summary of failed tests is showing in coverage file list but it's not a big deal

@charlesBochet Should we include the coverage report?

**Comment by charlesBochet** [comment]
Not in the test command, you are right

File: packages/twenty-front/jest.config.ts [hunk 1]
```diff
@@ -19,7 +21,6 @@ export default {
       functions: 60,
     },
   },
-  collectCoverage: true,
```
Comment (gitstart-twenty) [inline packages/twenty-front/jest.config.ts:22]: Updated this to be consistent across the board, coverage should be disabled by default.

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**